src/components/layout.css

```diff
@@ -263,3 +263,7 @@ table {
   text-align: justify;
   grid-area: excerpt;
 }
+
+.active{
```

Comment:
Hello,

I suggest to remove this "global" class `.active` and follow BEM methodology.

Since we've a class named `.site-header__logo-link` and active is a modifier, you should use: `.site-header__logo-link--active`

Reply:
Followed the BEM methodology, and done the required changes